src/Symfony/Component/Translation/Tests/Command/TranslationPullCommandTest.php

```diff
@@ -740,12 +785,13 @@ private function createCommand(ProviderInterface $provider, array $locales = ['e
         $reader->addLoader('yml', new YamlFileLoader());
 
         return new TranslationPullCommand(
-            $this->getProviderCollection($provider, $providerNames, $locales, $domains),
-            $writer,
-            $reader,
-            $defaultLocale,
-            [$this->translationAppDir.'/translations'],
-            $locales
+            providerCollection: $this->getProviderCollection($provider, $providerNames, $locales, $domains),
+            writer: $writer,
+            reader: $reader,
+            defaultLocale: $defaultLocale,
+            transPaths: [$this->translationAppDir.'/translations'],
+            enabledLocales: $locales,
+            eventDispatcher: $dispatcher
```

Comment:
As Symfony does not provide BC for argument names (except for attribute constructors), I would not use them in tests: this would hide BC breaks changing the parameter list by adding a new parameter in the middle.